packages/javascript/bh-shared-ui/src/components/InfiniteScrollingTable/InfiniteScrollingTable.tsx

```diff
@@ -49,7 +49,7 @@ const createItemData = memoize((items, onClick) => ({
     onClick,
 }));
 
-const Row = memo(({ data, index, style }: ListChildComponentProps) => {
+const Row = memo(function Row({ data, index, style }: ListChildComponentProps) {
```

Comment:
TIL that a named function resolves the missing display name error! Up until now, I would just add Component.displayName = "Component" hah thanks!

Reply:
Yep, a new linting rule was added to check for these anonymous components.